### PR TITLE
strange focus bug

### PR DESCRIPTION
**Pull request by itay1313**

### CODE REVIEW

**Comment by itay1313** [comment]
[DX-103](https://jira.liveperson.com/browse/DX-103) 
and
[DX-100](https://jira.liveperson.com/browse/DX-100)

File: _includes/sidebar.html [hunk 1]
```diff
@@ -166,4 +166,13 @@
       </span>
     </a>
   </li>
+  <li class="mobile-sign-in">
+    <a
+      target="_blank"
+      href="https://authentication.liveperson.net/login.html"
+      class="loginText"
+    >
+      <span>Sign in</span>
```
Comment (j9t) [inline _includes/sidebar.html:175]: Sanity check, is the `span` needed?

File: css/inner-page.css [hunk 2]
```diff
@@ -434,6 +434,14 @@ ul#mysiderbar li {
     padding-left: 11px;
 }
 
+.mysidebar>li.mobile-sign-in {
+    padding: 0;
+}
+
+.mysidebar>li.mobile-sign-in a.loginText:hover {
+    background: white;
```
Comment (j9t) [inline css/inner-page.css:442]: We may not have a standard yet, but how about we work with hex here (`#fff`)?

**Review by j9t — CHANGES_REQUESTED** [review]

File: css/inner-page.css (hunk 2 above)
Comment (itay1313) [reply css/inner-page.css]: done

File: _includes/sidebar.html (hunk 1 above)
Comment (itay1313) [reply _includes/sidebar.html]: deleted it

**Review by j9t — APPROVED** [review]

**Pull request merged**